schemas/Signup.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "object",
+  "description": "Triggered when a new user accepts the analytics prompt that is shown during the registration flow.",
+  "properties": {
+    "eventName": {
+      "enum": ["Signup"]
+    },
+    "authenticationType": {
+      "type": "string",
+      "description": "The type of authentication that was used to sign up.",
+      "oneOf": [
+        {"const": "Password", "description": "Registration with a username and password."},
+        {"const": "SignInWithApple", "description": "Social login using Apple."},
+        {"const": "SignInWithFacebook", "description": "Social login using Facebook."},
+        {"const": "SignInWithGitHub", "description": "Social login using GitHub."},
+        {"const": "SignInWithGitLab", "description": "Social login using GitLab."},
+        {"const": "SignInWithGoogle", "description": "Social login using Google."},
+        {"const": "SSO", "description": "Registration using another SSO provider."}
+      ],
+      "required": true
+    }
+  },
+  "required": ["eventName"],
```

Comment:
also shouldn't the required attribute be specified here rather than up there?

Reply:
Yes good point. I did notice this when I opened the PR, but saw it done both ways in other events. That said updating it I noticed that quicktype doesn't honour it for TS so we should probably fix any `"required": true` properties.

Reply:
Done ✅